lua/better_escape.lua

```diff
@@ -7,7 +7,9 @@ local settings = {
     timeout = vim.o.timeoutlen,
     mappings = {
         i = {
+        --  startkey[s]
```

Comment:
perhaps first_key would be even clearer?

Reply:
Yep. It's much clearer.